app/lib/submission_builder/ty2022/states/ny/documents/it201.rb

```diff
@@ -59,27 +59,27 @@ def document
                 xml.TOT_NRFNDCR_AMT claimed: calculated_fields.fetch(:IT201_LINE_43)
                 xml.TX_AFT_NRFNDCR_AMT claimed: calculated_fields.fetch(:IT201_LINE_44)
                 xml.TOT_TX_AMT claimed: calculated_fields.fetch(:IT201_LINE_46)
-                xml.NYC_TXBL_INC_AMT claimed: calculated_fields.fetch(:IT201_LINE_47)
```

Comment:
Totally minor observation: If you wanted, you could also move the call to calculated_fields into the new method and just pass in the line symbol as the second argument

Reply:
Excellent Suggestion - Implemented!